raiden-ts/src/raiden.ts

```diff
@@ -358,7 +369,14 @@ export class Raiden {
         `Mismatch between provided account and loaded state: "${address}" !== "${loadedState.address}"`,
       );
 
-    const raiden = new Raiden(provider, network, signer, contracts, loadedState);
+    const raidenConfig: RaidenConfigDefaults = {
+      matrixServerLookup: '',
+      settleTimeout: 60000,
+      revealTimeout: 60000,
+      ...config
```

Comment:
These defaults need to be changed

Reply:
These timeouts are in blocks, not seconds. Defaults are `settleTimeout: 500`, `revealTimeout: 50` (as per today)